docs/asyncapi.yaml

```diff
@@ -620,6 +654,10 @@ components:
       format: uuid
       description: GUID for a specific event reference
       example: 931e1a76-cff2-43ef-bfe5-82742eeeae68
+    RemainingAmount:
+      type: number
+      description: Amount left to be paid
+      example: 750
     SBI:
```

Comment:
SBI is not Enum

Reply:
SBI:
      type: number
      description: Single Business Identifier
      minimum: 105000000
      maximum: 999999999
      example: 123456789
      
      SBI is given type: number